### DISCUSSION TITLE
number of towns difficulty

### DISCUSSION THREAD

**Discussion by Hollikill** [voxeltycoon/issues#496] [General]
add a difficulty setting for the number of settlements generated in the starting area

**Comment by katepolak** [comment]
I don't really see how this would be beneficial to be adjustable to be honest. The number of town is chosen to be able to sell most of the basic products without having to wait for the respective demands to spawn, doing the opposite would be a really bad idea as the player would be forced to do nothing and wait which isn't exactly the definition of fun.

But if you have a good reason for it, it will definitely be considered.

Also please describe you future suggestions in more detail, one sentence is simply not enough.

**Comment by Hollikill** [comment]
ah, i see how it is supposed to work now, makes sense. I just thought it would be a neat feature, there wasn't really a complicated reason for it. I will provide reasons for any further suggestions if I have any though!

**Comment by victorleblais** [comment]
I can see the benefit of having such an option.

I find that the starting region quickly becomes crowded with 3 towns (and with some seed you have 4 starting cities). I would love to have the option to have 1 or 2 cities in my starting area (even if it makes the start of the game slower).

Also, I would like it if not all additional regions had a city or resources (and price those region accordingly) Sometimes I just want more space :P

I guess all of that could also be fixed by a parameter like "region size" that changes the average surface area of a region...

**Comment by Sliderpro** [comment]
Come on, you can just reroll the world seed and start anew...
I can see where this is coming from, but imo 3-4 towns is more or less optimal number. 2 towns wont develop fast enough to force you to do research. 3-4 towns will (if you supply) inevitably force you to either develop or expand. 2 towns is a very slow and boring start with zero motivation to build any sort of complex network.

**Comment by victorleblais** [comment]
Just as a note to this conversation, the 0.77 came out recently and it added a few settings in order to change how the world generates (amount of mountains, lakes, plants, ...).

The most notable feature is the ability to change the size of regions, which greatly impact the "difficulty" of the starting area, but also solves "my issue" (I found that cities where too close to each other, so increasing region size helps a lot).
